Utils/template.nf

```diff
@@ -0,0 +1,22 @@
+params.tool.command.optional = ''
```

Comment:
Ik zou dit in de process config verwerken

Reply:
Dit zorgt ervoor dat je niet altijd een 'optional' moet meegeven in je process config. Dus alleen als je hem nodig hebt, dan kan je '' overschrijven in je config.